docker/backend.dockerfile

```diff
@@ -4,8 +4,6 @@ WORKDIR /app
 
 COPY ./server /app/server
 
-COPY .flaskenv /app
```

Comment:
The flaskenv is crucial for the backend to run

Reply:
See docker-compose.yml we are already doing it the proper way. This is not the proper way.